lms/views/application_instances.py

```diff
@@ -28,6 +28,7 @@ def create_application_instance(request):
         developer_key,
         developer_secret,
         request.registry.settings["aes_secret"],
+        canvas_sections_enabled=developer_key and request.feature("section_groups"),
```

Comment:
This logic can easily be moved into the model once it no longer depends on the feature flag